class AuthError(Exception):
    """Excepción personalizada para errores de autenticación."""
    pass
